Sources/KlaviyoSwift/NetworkSession.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  NetworkSession.swift
+//  Wrapper class for URLSession.
+//
+//  Created by Noah Durell on 11/3/22.
+//
+
+import Foundation
+
+private let defaultUserAgent = {
+    let info = Bundle.main.infoDictionary
+    let executable = (info?["CFBundleExecutable"] as? String) ??
+        (ProcessInfo.processInfo.arguments.first?.split(separator: "/").last.map(String.init)) ??
+        "Unknown"
+    let bundle = info?["CFBundleIdentifier"] as? String ?? "Unknown"
+    let appVersion = info?["CFBundleShortVersionString"] as? String ?? "Unknown"
+    let appBuild = info?["CFBundleVersion"] as? String ?? "Unknown"
+
+    let osNameVersion: String = {
+        let version = ProcessInfo.processInfo.operatingSystemVersion
+        let versionString = "\(version.majorVersion).\(version.minorVersion).\(version.patchVersion)"
+        let osName: String = "iOS"
+
+        return "\(osName) \(versionString)"
+    }()
+
+    let klaivyoSDKVersion = "klaviyo-ios/\(version)"
+
+    return "\(executable)/\(appVersion) (\(bundle); build:\(appBuild); \(osNameVersion)) \(klaivyoSDKVersion)"
```

Comment:
can we easily see % by sdk version this way? or should be have individual headers for each data point

Reply:
In splunk it should show up.
Happy to split it up too...we hadn't decided on names yet. Also I was planning on potentially including a lot of the same  in profile updates and events.